### PR TITLE
Initial metrics work for the indexer

### PR DESCRIPTION
**Pull request by dariorussi**
## Description 

Initial work for monitoring values. 
This is a good start for values to track and please advise on what else you'd like to see.

## Test plan 

Not much now and need to come up with something to check that things are good

---

## Release notes

Check each box that your changes affect. If none of the boxes relate to your changes, release notes aren't required.

For each box you select, include information after the relevant heading that describes the impact of your changes that a user might notice and any actions they must take to implement updates. 

- [ ] Protocol: 
- [ ] Nodes (Validators and Full nodes): 
- [ ] Indexer: 
- [ ] JSON-RPC: 
- [ ] GraphQL: 
- [ ] CLI: 
- [ ] Rust SDK:

### CODE REVIEW

File: crates/sui-bridge-indexer/src/main.rs [hunk 1]
```diff
@@ -54,19 +51,24 @@ async fn main() -> Result<()> {
...
-    // start metrics server
     let (_exit_sender, exit_receiver) = oneshot::channel();
-    let metrics = DataIngestionMetrics::new(&Registry::new());
 
     // Init metrics server
-    let metrics_address = SocketAddr::new(IpAddr::V4(Ipv4Addr::new(0, 0, 0, 0)), 1000);
-    let registry_service = start_prometheus_server(metrics_address);
-    let prometheus_registry = registry_service.default_registry();
-    mysten_metrics::init_metrics(&prometheus_registry);
-    info!("Metrics server started at port {}", 1000);
```
Comment (dariorussi) [inline crates/sui-bridge-indexer/src/main.rs:69]: reworked this initialization which I am not sure about and would love someone to take a look and comment if there are issues

File: crates/sui-bridge-indexer/src/worker.rs [hunk 2]
```diff
@@ -64,16 +73,19 @@ impl BridgeWorker {
...
+                    Self::process_sui_event(ev, tx, checkpoint, timestamp_ms, &self.metrics)?
+                {
                     result.push(data);
                 }
                 Ok::<_, anyhow::Error>(result)
             })?;
 
             if !token_transfers.is_empty() {
-                debug!(
+                info!(
```
Comment (patrickkuo) [inline crates/sui-bridge-indexer/src/worker.rs:88]: having individual event process log at info level can make the indexer quite chatty, is that ok?

**Review by patrickkuo — APPROVED** [review]
LGTM

File: crates/sui-bridge-indexer/src/worker.rs (hunk 2 above)
Comment (dariorussi) [reply crates/sui-bridge-indexer/src/worker.rs:88]: yeah we had that conversation and we decided to go with a more chatty interface for now. We can revisit if we end up with too much logging.
Thanks

**Pull request merged**